experimental/ModelBuilder/test/CMakeLists.txt

```
@@ -0,0 +1,275 @@
+    test-simple-jit-vulkan
+    test-simple-mlir
+    test-vec-to-gpu
+    test-vector-transfers-jit
+    iree::tools::IreeFileCheck
+)
+
+iree_cc_binary(
+  NAME
+    test-dot-prod
+  OUT
+    test-dot-prod
+  SRCS
+    "TestDotProdJIT.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-vector-transfers-jit
+  OUT
+    test-vector-transfers-jit
+  SRCS
+    "TestVectorTransfersJIT.cpp"
+  DEPS
+    runtime-support.so
+    LLVMSupport
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-mnist-jit
+  OUT
+    test-mnist-jit
+  SRCS
+    "TestMNISTJIT.cpp"
+  DEPS
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    MLIRmlir_runner_utils
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-simple-jit
+  OUT
+    test-simple-jit
+  SRCS
+    "TestSimpleJIT.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIREDSC
+    MLIRIR
+    MLIRSCFTransforms
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+)
+
+iree_cc_binary(
+  NAME
+    test-simple-jit-vulkan
+  OUT
+    test-simple-jit-vulkan
+  SRCS
+    "TestSimpleJITVulkan.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIRIR
+    MLIRParser
+    MLIRSPIRV
+    MLIRmlir_runner_utils
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+    iree::base::initializer
+    iree::hal::llvmjit::llvmjit_driver_module
+    iree::hal::vmla::vmla_driver_module
+    iree::hal::vulkan::vulkan_driver_module
+    vulkan-runtime-wrappers
+)
+
+iree_cc_binary(
+  NAME
+    test-matmul-vulkan
+  OUT
+    test-matmul-vulkan
+  SRCS
+    "TestMatMulVulkan.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIRExecutionEngine
+    MLIRGPU
+    MLIRGPUToSPIRVTransforms
+    MLIRGPUToVulkanTransforms
+    MLIRIR
+    MLIRLinalgOps
+    MLIRLinalgToLLVM
+    MLIRLinalgTransforms
+    MLIRParser
+    MLIRPass
+    MLIRSPIRV
+    MLIRStandardToLLVM
+    MLIRStandardToSPIRVTransforms
+    MLIRSupport
+    MLIRTargetLLVMIR
+    MLIRTransformUtils
+    MLIRVectorToLLVM
+    MLIRmlir_runner_utils
+    experimental::ModelBuilder
+    experimental::ModelBuilder::ModelRunner
+    experimental::ModelBuilder::VulkanLaunchWrapper
+    iree::base::initializer
+    iree::compiler::Conversion::LinalgToSPIRV
+    iree::hal::llvmjit::llvmjit_driver_module
+    iree::hal::vmla::vmla_driver_module
+    iree::hal::vulkan::vulkan_driver_module
+    vulkan-runtime-wrappers
+)
+
+iree_cc_binary(
+  NAME
+    test-vec-to-gpu
+  OUT
+    test-vec-to-gpu
+  SRCS
+    "TestVectorToGPU.cpp"
+  DEPS
+    LLVMSupport
+    MLIRAllDialects
+    MLIRExecutionEngine
+    MLIRGPU
+    MLIRGPUToVulkanTransforms
```

```suggestion
    MLIRGPUtoVulkanTransforms
```
Same here as long as the LLVM submodule isn't bumped.

---

Done.